src/main/resources/flows-config.yaml

```
@@ -85,7 +85,7 @@ flow:
       - name: parent-partner-info-disability
   parent-partner-info-disability:
     nextScreens:
-      - name: parent-other-family
+      - name: parent-intro-family-info
```

This page is now in main
```suggestion
      - name: parent-other-family
```